dashboard/app/controllers/projects_lists_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ProjectsListsController < ApplicationController
+  include LevelsHelper
+
+  def index
+    section_id = params[:section_id]
+    section_owner_id = Section.find(section_id).user_id
+    raise "section can only be accessed by its owner" unless current_user && current_user.id == section_owner_id
```

Comment:
Hey @aoby, the CanCan step seems to work when I add `load_and_authorize_resource :section`, but only when the placeholder in the route is named `:section_id`, i.e. `get "/:section_id", to: 'projects_lists#index'`. Why is this? I don't see any hints in the documentation at https://github.com/ryanb/cancan/wiki/authorizing-controller-actions, or http://www.rubydoc.info/github/ryanb/cancan/CanCan/ControllerAdditions/ClassMethods, or http://guides.rubyonrails.org/routing.html.